app/ios/AriesBifold/Info.plist

```diff
@@ -63,7 +63,7 @@
 	<key>NSFaceIDUsageDescription</key>
 	<string>$(PRODUCT_NAME) wants to use your FaceID/TouchID to authenticate your identity</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string></string>
+	<string>Our app does not request this permission or utilize this functionality but it is included in our Info.plist since our app utilizes the react-native-permissions library, which references this permission in its code.</string>
```

Comment:
Kind of technical for a user facing explanation. How about:

```
We include this permission because a library our app uses references it, even though we don't use it directly.
```

Reply:
done